packages/client/src/profile/serviceApi.ts

```diff
@@ -54,7 +53,7 @@ function request<T>(options: AxiosRequestConfig) {
 
 const sendVerifyCode = (data: ISendVerifyCodeData) => {
   return request<ISendVerifyCodeResponse>({
-    url: resolve(window.config.API_GATEWAY_URL, 'sendVerifyCode'),
+    url: new URL(window.config.API_GATEWAY_URL, 'sendVerifyCode').toString(),
```

Comment:
The parameters should be other way around. With `new URL` the first one is the path, and the second one is the base URL.

Reply:
Thanks Pyry! I must have missed this one